### PR TITLE
CRAYSAT-1726: Add information about multitenancy

### PR DESCRIPTION
**Pull request by jack-stanek-hpe**
## Summary and Scope
This change adds information on how the tenant name can be configured and links it from the installation instructions.

## Issues and Related PRs

* Resolves [CRAYSAT-1726](https://jira-pro.it.hpe.com:8443/browse/CRAYSAT-1726)

### Tested on:

  * Local development environment

### Test description:
Build docs and check for correctness.

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: .spelling [hunk 1]
```diff
@@ -64,6 +64,7 @@ hostname
 hostnames
 metaschema
 metadata
+multitenancy
```
Comment (haasken-hpe) [inline .spelling:67]: I'm going to suggest that we use "multi-tenancy" and "multi-tenant" with hyphens. This is consistent with Jeremy's 2023 CUG paper (and the references) on the topic.

File: docs/install.md [hunk 2]
```diff
@@ -225,6 +225,12 @@ Information](#set-system-revision-information)).
    the configuration files must be copied from `ncn-m001` to `ncn-m002` and
    `ncn-m003`. Therefore, the list of hosts above is `ncn-m002` and `ncn-m003`.
 
+### (Optional) Configure multitenancy
```
Comment (haasken-hpe) [inline docs/install.md:228]: It looks like we're using title case here, so this should be:

```suggestion
### (Optional) Configure Multi-tenancy
```

File: docs/install.md [hunk 3]
```diff
@@ -225,6 +225,12 @@ Information](#set-system-revision-information)).
...
+
+If SAT is being installed on a multitenant system, the tenant name can be
```
Comment (haasken-hpe) [inline docs/install.md:230]:
```suggestion
If SAT is being installed on a multi-tenant system, the tenant name can be
```

File: docs/install.md [hunk 4]
```diff
@@ -225,6 +225,12 @@ Information](#set-system-revision-information)).
...
+configured at this point. For more information, see [Configure
+multitenancy](usage/multitenancy.md).
```
Comment (haasken-hpe) [inline docs/install.md:232]:
```suggestion
multi-tenancy](usage/multi-tenancy.md).
```

File: docs/usage/README.md [hunk 5]
```diff
@@ -3,3 +3,4 @@
 - [SAT Bootprep](sat_bootprep.md)
 - [SAT and IUF](sat_and_iuf.md)
 - [Change the BOS Version](change_bos_version.md)
+- [Configure multitenancy](multitenancy.md)
```
Comment (haasken-hpe) [inline docs/usage/README.md:6]:
```suggestion
- [Configure Multi-tenancy](multi-tenancy.md)
```

File: docs/usage/multitenancy.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Configure multitenancy
+
+SAT supports supplying tenant information to CSM services in order to allow
+tenant admins to use SAT within their tenant.
+
+By default, the tenant name is not set, and SAT will not send any tenant
+information with its requests to CSM services.
+
+## Configuring the tenant name in the SAT configuration file
```
Comment (haasken-hpe) [inline docs/usage/multitenancy.md:9]:
```suggestion
## Configuring the Tenant Name in the SAT Configuration File
```

File: docs/usage/multitenancy.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+`api_gateway.tenant_name` option.
+
+For example:
+
+```toml
+[api_gateway]
+tenant_name = "my_tenant"
+```
+
+## Configuring the tenant name on the command line
```
Comment (haasken-hpe) [inline docs/usage/multitenancy.md:21]:
```suggestion
## Configuring the Tenant Name on the Command Line
```

**Review by haasken-hpe — APPROVED** [review]
Minor comments. Generally, use title case in headings for consistency with existing docs, and use hyphenated "multi-tenancy" and "multi-tenant".

**Review by jld-dehn-hpe — APPROVED** [review]

**Pull request merged**